lib/src/visitors/expression_parser_visitor.dart

```diff
@@ -15,7 +15,13 @@ class ExpressionParserVisitor<C> implements TemplateAstVisitor<Null, C> {
       : exceptionHandler = exceptionHandler ?? new ThrowingExceptionHandler();
 
   @override
-  visitAttribute(AttributeAst astNode, [_]) => null;
+  visitAttribute(AttributeAst astNode, [_]) {
+    if (astNode.mustaches != null) {
+      astNode.mustaches.forEach((mustache) {
```

Comment:
Won't be able to change these (as discussed in person) because 1.21 likes to complain when using this.